hack/deploy-cluster.sh

```diff
@@ -74,4 +74,40 @@ aks-engine deploy --subscription-id ${SUBSCRIPTION_ID} \
   --client-id ${CLIENT_ID} \
   --client-secret ${CLIENT_SECRET}
 echo "Kubernetes cluster deployed. Please find the kubeconfig for it in _output/"
+export KUBECONFIG=_output/kubeconfig/kubeconfig.$LOCATION.json
 
+#Deploy AzureDisk CSI Plugin
+COMMAND_DISK="kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/master/deploy"
+config_disk=("crd-csi-driver-registry.yaml" "crd-csi-node-info.yaml" "rbac-csi-attacher.yaml" "rbac-csi-driver-registrar.yaml" "rbac-csi-provisioner.yaml" "rbac-csi-snapshotter.yaml" "csi-azuredisk-provisioner.yaml" "csi-azuredisk-attacher.yaml" "azuredisk-csi-driver.yaml" "example/storageclass-azuredisk-csi.yaml")
+for i in {1..10}
+do
+$COMMAND_DISK/$config_disk
+done
```

Comment:
the loop is not right, could we replace it from https://github.com/kubernetes/cloud-provider-azure/blob/master/docs/cloud-controller-manager.md#deploy-azuredisk-csi-plugin